app/src/main/java/com/skgtecnologia/sisem/data/remote/model/body/ButtonResponse.kt

```diff
@@ -12,7 +12,7 @@ import com.valkiria.uicomponents.props.ButtonStyle
 
 @JsonClass(generateAdapter = true)
 data class ButtonResponse(
-    @Json(name = "identifier") val identifier: String?,
```

Comment:
Revert this one, we should always have nullable DTOs to avoid crashes and propagate exceptions to be handled upstream